miranda/convert/_data_corrections.py

```diff
@@ -426,9 +461,12 @@ def metadata_conversion(d: xr.Dataset, p: str, m: Dict) -> xr.Dataset:
     return d
 
 
-def variable_conversion(ds: xr.Dataset, project: str) -> xr.Dataset:
+def variable_conversion(
+    ds: xr.Dataset, project: str, files: Optional[List[str]] = None
```

Comment:
What is the purpose of `files` here?

Reply:
It was an idea I abandoned to address an issue with data, but the problem demands fixes on open, so not appropriate at this step. removed.